.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Install tox
+        run: python -m pip install tox
+      - name: Run linting
+        run: python -m tox -e lint
+
+  test:
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8]
+        platform: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python }}
+      - name: "Install dependencies"
+        run: |
+          python -m pip install --upgrade pip setuptools wheel
+          python -m pip install --upgrade coverage[toml] virtualenv tox
+      - name: Run tests
+        run: python -m tox -e py  # Run tox using the version of Python in `PATH`
+      - name: "Convert coverage"
+        run: "python -m coverage report"
```

Comment:
I don't think this is necessary, because it's already printed during "Run tests" via the `--cov-report` option:

https://github.com/callmecampos/twine/runs/751356327?check_suite_focus=true#step:5:65

```suggestion
```